### PR TITLE
RESTWS-783: upgrade swagger-core from 1.5.13 to 1.6.2

### PR DESCRIPTION
**Pull request by achilep**
## Issue I worked on

see https://issues.openmrs.org/browse/RESTWS-783

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by achilep** [comment]
Screenshot from 2020-06-27 23-52-30

**Comment by sherrif10** [comment]
Thanks for this Did you first check on the swagger ,
LGTM cc @dkayiwa @mozzy11

File: omod-common/pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		<dependency>
 			<groupId>io.swagger</groupId>
 			<artifactId>swagger-core</artifactId>
-			<version>1.5.13</version>
+			<version>1.6.1</version>
```
Comment (mozzy11) [inline omod-common/pom.xml:36]: wel done @achilep .

Is this the latest version really ??
https://mvnrepository.com/artifact/io.swagger/swagger-core

Comment (achilep) [reply omod-common/pom.xml:36]: thanks @mozzy11  i will use the 1.6.2 version

File: omod-common/pom.xml [hunk 2]
```diff
@@ -33,7 +33,7 @@
 		<dependency>
 			<groupId>io.swagger</groupId>
 			<artifactId>swagger-core</artifactId>
-			<version>1.5.13</version>
+			<version>1.6.2</version>
```
Comment (sherrif10) [inline omod-common/pom.xml:36]: LGTM   cc @dkayiwa

Comment (mozzy11) [reply omod-common/pom.xml:36]: @achilep   did you these changes with the reff app ??

**Comment by achilep** [comment]
Screenshot from 2020-08-21 07-40-10

**Pull request merged**